flink-clients/src/test/java/org/apache/flink/client/cli/util/MockedCliFrontend.java

```diff
@@ -20,40 +20,22 @@
 
 import org.apache.flink.client.cli.CliFrontend;
 import org.apache.flink.client.cli.CliFrontendTestUtils;
-import org.apache.flink.client.cli.CustomCommandLine;
 import org.apache.flink.client.program.ClusterClient;
 import org.apache.flink.configuration.Configuration;
 
-import org.apache.commons.cli.CommandLine;
-
-import static org.mockito.Matchers.any;
-import static org.mockito.Matchers.anyString;
-import static org.powermock.api.mockito.PowerMockito.mock;
-import static org.powermock.api.mockito.PowerMockito.when;
+import java.util.Collections;
 
 /**
  * Utility class for mocking the {@link ClusterClient} within a {@link CliFrontend}.
  *
  * <p>The mocking behavior can be defined in the constructor of the sub-class.
  */
 public class MockedCliFrontend extends CliFrontend {
```

Comment:
nit: Name is not ideal because it's not required that only mocks are passed as a constructor argument.

Reply:
True. But since I didn't introduce it, I'll keep it like it is (avoiding further downstream merge conflicts).